doc/api-extensions.md

```diff
@@ -2437,7 +2437,7 @@ Adds the ability to convert images from different formats (e.g. VMDK or QCow2) i
 
 ## `instance_create_start`
 
-Adds a new `start` field to the `POST /1.0/instances` API which when set
+Adds a `start` field to the `POST /1.0/instances` API which when set
```

Comment:
Please can you reword this in https://github.com/canonical/lxd/pull/13695/commits/00b07d1b6468dbc15e34f8eeb0a12bc9e5e33462 and keep the doc change below as a separate commit

Reply:
Done, thanks